src/main/java/com/org/jenkins/custom/jenkins/distribution/service/GithubController.java

```diff
@@ -0,0 +1,56 @@
+package com.org.jenkins.custom.jenkins.distribution.service;
+
+import java.util.List;
+import org.kohsuke.github.GHApp;
+import org.kohsuke.github.GHAppInstallation;
+import org.kohsuke.github.GHAppInstallationToken;
+import org.kohsuke.github.GitHub;
+import org.kohsuke.github.GitHubBuilder;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.core.env.Environment;
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RestController;
+
+import static com.org.jenkins.custom.jenkins.distribution.service.GithubJwtHelper.createJWT;
+import static com.org.jenkins.custom.jenkins.distribution.service.GithubJwtHelper.get;
+
+
+@RestController
+@CrossOrigin("*")
+public class GithubController {
+
+    @Value("${APP_IDENTIFIER}")
+    private String getAppIdentifier;
+
+    @PostMapping(path = "/event_handler")
+    public void handleEvent(@RequestBody String requestBodyString) throws Exception {
+        System.out.println("Received event handler event");
```

Comment:
Should the logger be used here instead of `System.out`?

Reply:
Oh yeah my bad :(